cargo-near/src/commands/build_command/docker.rs

```diff
@@ -83,64 +89,61 @@ impl super::BuildCommand {
         docker_build_meta: metadata::ReproducibleBuild,
         cloned_repo: &cloned_repo::ClonedRepo,
     ) -> color_eyre::eyre::Result<(ExitStatus, Command)> {
-        // Cross-platform process ID and timestamp
-        let pid = id().to_string();
-        let timestamp = SystemTime::now()
-            .duration_since(UNIX_EPOCH)
-            .unwrap()
-            .as_secs()
-            .to_string();
+        let mut docker_cmd: Command = {
+            // Platform-specific UID/GID retrieval
+            #[cfg(unix)]
+            let uid_gid = format!("{}:{}", getuid(), getgid());
```

Comment:
It should be Linux-specific since we don't need to match the uid/gid on MacOS or other *nixes. Please, update the comment to highlight the reason for this mapping is that on Linux the volume is mounted natively and thus the unprivileged user inside Docker container should be able to write to the mounted folder that has the host user permissions

Reply:
fixed by https://github.com/near/cargo-near/pull/159/commits/d8598dff6172b96c18b9db2135407e7d67362c89